.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=11111111
+MYSQLDB_DATABASE=books
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8081
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
Don’t  push your `.env` file to GitHub. It’s not security, add it to .gitignore. 
You should add `env.sample `and leave variables without values 
Example:
MYSQLDB_USER= #fill your username 
MYSQLDB_ROOT_PASSWORD=#fill your password 
....etc

Reply:
i just add it for example

Reply:
yep